📝 Add CHANGELOG (#568)
likecoin/liker-land: CHANGELOG.md

# Changelog

## 2022-08-24
306e74e2a565f7ec352560ca9539cb1e95ff2cfa

### Added

> - Transfer feature on the details page that allows users to send their own NFTs to any like1 address.

Reviewer: ```suggestion
- Allow users to transfer their NFTs.
```

> - Share button to easily share NFT and portfolio pages.

Reviewer: ```suggestion
- Added a share button on the details page and the portfolio page.
```

### Changed

> - Improve the collect flow.

Reviewer 2: Integrate the collect flow into liker.land instead of using popup

# Changelog

## 2022-08-24

> df0c367f021a6ced7b9039cbcb580f7ab33e9a42

Reviewer: ```suggestion
306e74e2a565f7ec352560ca9539cb1e95ff2cfa
```

# Changelog

## 2022-08-24
df0c367f021a6ced7b9039cbcb580f7ab33e9a42

### Added

> - Users can collect works on liker.land, no need to redirect to app.like.co.

Author: Or maybe just say we improved the collect flow?

Reviewer: Yes, I think simple is better

I think should use Changed instead of Added

Ref: https://keepachangelog.com/en/1.0.0/

- Introduce a transfer feature on the details page that allows users to send their own NFTs to any like1 address.

> - Easily share NFT and portfolio pages by the share button.

Reviewer: Added

> - Provides four login methods on the cosmos chain (Keplr Extension, Keplr Mobile, Wallet Connect, Cosmostation)

Reviewer: Do not have to mention yet
